### DISCUSSION TITLE
How to use highcharts dashboards?

### DISCUSSION THREAD

**Discussion by Antonio-Serrat** [richardeschloss/nuxt-highcharts#76] [Q&A]
I'm trying to use dashboards but I dont know how implement it. 

I want to use something like that -> https://www.highcharts.com/demo/dashboards/minimal 
but only with one line chart type

Anyone can help me?

**Comment by richardeschloss** [comment]
That's a pretty cool new feature. Thanks for bringing it to my attention. I'll have to check it out. Seems to be a game changer.
For Nuxt, we might want to consider [Dashboards For Vue](https://www.highcharts.com/docs/dashboards/dashboards-with-vue). 

Here's a working example: (that only needs to use Highcharts plugins, doesn't need any special Vue plugins)
https://stackblitz.com/edit/dashboards-vue3-fcutdg?file=src%2FDashboard.vue

Perhaps a convenience, nuxt-highcharts could list "@highcharts/dashboards" as an optional dependency and assist with some of the boilerplate stuff, but I think for now I'll hold off that implementation since the feature is still new.

**Reply by Antonio-Serrat** [reply]
You're welcome! that it's the community :D 

Thank you for the example, but I can see it. 
![image](https://github.com/user-attachments/assets/30634c3c-ca7b-4e6d-b7ae-a5a9141cf734)
![image](https://github.com/user-attachments/assets/df1420f4-3f3a-4bb2-be66-bbeca87d1346)

Anyway, I'm going to use Highcharts and a simple table below to display the data as well and then try to bind hovering over the points to highlight the row in the table and vice versa.


I tell you if this works!
